i18n/ja/docusaurus-plugin-content-docs/current/guides/mobile-plugin-ima-ios.md

```diff
@@ -0,0 +1,51 @@
+---
+title: UID2 IMA Plugin for iOS
+description: A guide for publishers using IMA for ad requests on iOS apps.
+hide_table_of_contents: false
+sidebar_position: 15
+---
+
+# UID2 IMA Plugin for iOS Integration Guide
+
+The UID2 Interactive Media Ads (IMA) Plugin for iOS enables publishers that use the [Google Interactive Media Ads SDK](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/) to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288?hl=en-GB) in ad requests from iOS apps. This is done automatically, with no direct coding needed from the app developer.
+
+## Functionality
+
+This plugin simplifies integration with Google Interactive Media Ads (IMA) for any publishers who want to support UID2 for apps running on iOS devices.
+
+## Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the UID2 IMA Plugin for iOS version 0.2.0.
+
+## GitHub Repository
+
+This plugin is in the following open-source GitHub repository:
+
+- [https://github.com/IABTechLab/uid2-ios-plugin-google-ima](https://github.com/IABTechLab/uid2-ios-plugin-google-ima)
+
+## Requirements 
+
+To run this plugin, install the following:
+
+1. [Google IMA SDK v3.19.1](https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/history)
+1. [UID2 SDK for iOS v0.2.0](../sdks/uid2-sdk-ref-ios.md)
+1. UID2 IMA Plugin for iOS v0.2.0
+
+## Installation
+
+Prerequisite: Install the Google Interactive Media Ads SDK and the UID2 iOS SDK.
+
+Install the UID2 iOS IMA Plugin v0.2.0 via Swift Package Manager to an existing app with the UID2 iOS SDK and the Google Interactive Media Ads SDK installed.
+
+```
+.package(url: https://github.com/IABTechLab/uid2-ios-plugin-google-ima.git, exact: "0.2.0")
+```
+
+## Troubleshooting 
+
+To resolve any issue that might arise, it's recommended that you copy the two source code files from the UID2 IMA Plugin directly into your app. Be sure to use the same class and file names, to ensure that the Google IMA SDK can find the files at runtime. The two files are:
```

Comment:
I don't think we should include this.  While it is generally true, we do not want to encourage people to integrate directly like this as it'll be more difficult for them to upgrade over time.  If there is no other content for Troubleshooting then we should just remove this entire section.

Reply:
I see this content is also in the GMA iOS doc.  We should remove it from that doc too.

Reply:
Done for both -- thx.